### PR TITLE
Disabled getty1, Using TTYReset=no in getty service

### PR DESCRIPTION
**Pull request by moul**

### CODE REVIEW

File: 15.04/patches/etc/systemd/system/serial-getty@.service [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+Before=getty.target
+IgnoreOnIsolate=yes
+
+[Service]
+ExecStart=-/sbin/agetty --keep-baud 115200,38400,9600 %I $TERM
+Type=idle
+Restart=always
+UtmpIdentifier=%I
+TTYPath=/dev/%I
+TTYReset=no
```
Comment (moul) [inline 15.04/patches/etc/systemd/system/serial-getty@.service]: This is the line I changed

File: 15.04/Dockerfile [hunk 2]
```diff
@@ -106,12 +104,14 @@ RUN systemctl enable \
 	oc-gen-machine-id \
 	oc-kernel-check \
  && systemctl disable \
-	systemd-modules-load.service \
-	systemd-update-utmp-runlevel \
+	getty@tty1 \
```
Comment (moul) [inline 15.04/Dockerfile]: This is the newly disabled service

**Comment by moul** [comment]
```console
Welcome to Scaleway' initrd            http://j.mp/scw-initrd-linux

███████╗ ██████╗ █████╗ ██╗     ███████╗██╗    ██╗ █████╗ ██╗   ██╗
██╔════╝██╔════╝██╔══██╗██║     ██╔════╝██║    ██║██╔══██╗╚██╗ ██╔╝
███████╗██║     ███████║██║     █████╗  ██║ █╗ ██║███████║ ╚████╔╝
╚════██║██║     ██╔══██║██║     ██╔══╝  ██║███╗██║██╔══██║  ╚██╔╝
███████║╚██████╗██║  ██║███████╗███████╗╚███╔███╔╝██║  ██║   ██║
╚══════╝ ╚═════╝╚═╝  ╚═╝╚══════╝╚══════╝ ╚══╝╚══╝ ╚═╝  ╚═╝   ╚═╝

>>> Initrd build: v3.1-1-g9d38693 - 2015/08/25
>>> Kernel version: 4.0.5-235

>>> Installing busybox...
>>> Cloning initramfs for shutdown......
>>> Mounting pseudo-filesystems......
>>> Mounting /dev...
>>> Mounting /dev/pts...
>>> Mounting /run...
>>> Mounting /sys...
>>> Mounting /proc...

>>> Kernel cmdline: earlyprintk=ttyS0 console=ttyS0,9600n8 nousb noplymouth ip=dhcp boot=local root=/dev/nbd0 USE_XNBD=1 nbd.max_parts=8

>>> Configuring GPIOs...
>>> Configuring network...

>>> Server ID: c08c2251-6633-4e4d-8a50-c72d41e033b4
>>> Private IP: 10.1.32.129
>>> Public IP: 212.47.233.18
>>> Hostname: cocky-pike

>>> Checking user input for debug/verbose mode...
>>> Checking user input for debug shell...
>>> Signaling the server is kernel-started to the control plane...
>>> 'scw-update-server-state kernel-started' failed
>>> Adjusting time (ntp)...
>>> Initializing 'rescue' root file system...
>>> Mounting rescue root: http://test-images.fr-1.storage.online.net/scw-distrib-ubuntu-vivid.tar...
>>> Mounting /newroot...
[   26.320838] random: nonblocking pool is initialized
>>> Signaling the server is booted to the control plane...
>>> Creating default configuration files on rootfs......
>>> Checking metadata for debug shell...
>>> Checking metadata for debug sshd (dropbear)...
>>> Mounting /newroot/proc...
>>> Mounting /newroot/sys...
>>> Mounting /newroot/run...
>>> Mounting /newroot/dev...
>>> Moving initramfs backup to /run/initramfs for shutdown...
>>> Switching to linux...
End of Scaleway' initrd
           _ _       _      _          _ _
 ___ _ _ _|_| |_ ___| |_   | |_ ___   | |_|___ _ _ _ _
|_ -| | | | |  _|  _|   |  |  _| . |  | | |   | | |_'_|
|___|_____|_|_| |___|_|_|  |_| |___|  |_|_|_|_|___|_,_|

[   43.348631] systemd[1]: Failed to insert module 'autofs4'
[   43.417185] systemd[1]: systemd 219 running in system mode. (+PAM +AUDIT +SELINUX +IMA +APPARMOR +SMACK +SYSVINIT +UTMP +LIBCRYPTSETUP +GCRYPT -GNUTLS +ACL +XZ -LZ4 -SECCOMP +BLKID -ELFUTILS +KMOD -IDN)
[   43.633381] systemd[1]: Detected architecture arm.

Welcome to Ubuntu 15.04!

[   43.737173] systemd[1]: Set hostname to <cocky-pike>.
[  OK  ] Reached target Network is Online.
[   43.947072] systemd[1]: Reached target Network is Online.
[   44.011934] systemd[1]: Starting Network is Online.
[  OK  ] Reached target Remote File Systems (Pre).
[   44.136759] systemd[1]: Reached target Remote File Systems (Pre).
[   44.209969] systemd[1]: Starting Remote File Systems (Pre).
[  OK  ] Reached target Encrypted Volumes.
[   44.347402] systemd[1]: Reached target Encrypted Volumes.
[   44.412260] systemd[1]: Starting Encrypted Volumes.
[   44.470942] systemd[1]: Starting of Arbitrary Executable File Formats File System Automount Point not supported.
[  OK  ] Reached target Swap.
[   44.637070] systemd[1]: Reached target Swap.
[   44.688385] systemd[1]: Starting Swap.
[  OK  ] Created slice Root Slice.
[   44.787401] systemd[1]: Created slice Root Slice.
[   44.843933] systemd[1]: Starting Root Slice.
[  OK  ] Listening on Journal Socket (/dev/log).
[   44.957718] systemd[1]: Listening on Journal Socket (/dev/log).
[   45.028833] systemd[1]: Starting Journal Socket (/dev/log).
[  OK  ] Created slice User and Session Slice.
[   45.157718] systemd[1]: Created slice User and Session Slice.
[   45.226762] systemd[1]: Starting User and Session Slice.
[  OK  ] Listening on /dev/initctl Compatibility Named Pipe.
[   45.376759] systemd[1]: Listening on /dev/initctl Compatibility Named Pipe.
[   45.460374] systemd[1]: Starting /dev/initctl Compatibility Named Pipe.
[  OK  ] Listening on Delayed Shutdown Socket.
[   45.607723] systemd[1]: Listening on Delayed Shutdown Socket.
[   45.676751] systemd[1]: Starting Delayed Shutdown Socket.
[  OK  ] Created slice System Slice.
[   45.807402] systemd[1]: Created slice System Slice.
[   45.866035] systemd[1]: Starting System Slice.
[  OK  ] Created slice system-serial\x2dgetty.slice.
[   45.987402] systemd[1]: Created slice system-serial\x2dgetty.slice.
[   46.062683] systemd[1]: Starting system-serial\x2dgetty.slice.
[  OK  ] Listening on udev Control Socket.
[   46.197719] systemd[1]: Listening on udev Control Socket.
[   46.262594] systemd[1]: Starting udev Control Socket.
[  OK  ] Listening on udev Kernel Socket.
[   46.387071] systemd[1]: Listening on udev Kernel Socket.
[   46.450891] systemd[1]: Starting udev Kernel Socket.
[   46.512579] systemd[1]: Listening on Journal Audit Socket.
[   46.580537] systemd[1]: Starting Increase datagram queue length...
         Starting Increase datagram queue length...
[   46.717948] systemd[1]: Started Dispatch Password Requests to Console Directory Watch.
[   46.813030] systemd[1]: Starting Dispatch Password Requests to Console Directory Watch.
[  OK  ] Listening on Journal Socket.
[   46.976758] systemd[1]: Listening on Journal Socket.
[   47.036442] systemd[1]: Starting Journal Socket.
[   47.091961] systemd[1]: Started Create list of required static device nodes for the current kernel.
[   47.200745] systemd[1]: Started Set Up Additional Binary Formats.
[   47.275964] systemd[1]: Starting udev Coldplug all Devices...
         Starting udev Coldplug all Devices...
[   47.409721] systemd[1]: Starting SCW kernel requirements checker...
         Starting SCW kernel requirements checker...
[   47.549562] systemd[1]: Starting Create Static Device Nodes in /dev...
         Starting Create Static Device Nodes in /dev...
[   47.709000] systemd[1]: Mounting POSIX Message Queue File System...
         Mounting POSIX Message Queue File System...
[   47.849511] systemd[1]: Starting Load Kernel Modules...
         Starting Load Kernel Modules...
[   47.969726] systemd[1]: Starting Remount Root and Kernel File Systems...
         Starting Remount Root and Kernel File Systems...
[   48.129441] systemd[1]: Mounting Debug File System...
         Mounting Debug File System...
[  OK  ] Reached target Slices.
[   48.296756] systemd[1]: Reached target Slices.
[   48.350165] systemd[1]: Starting Slices.
[   48.397611] systemd[1]: Mounted Huge Pages File System.
[   48.460638] systemd[1]: Started Forward Password Requests to Wall Directory Watch.
[   48.551566] systemd[1]: Starting Forward Password Requests to Wall Directory Watch.
[  OK  ] Reached target Paths.
[   48.697402] systemd[1]: Reached target Paths.
[   48.749769] systemd[1]: Starting Paths.
[  OK  ] Mounted Debug File System.
[   48.847728] systemd[1]: Mounted Debug File System.
[  OK  ] Mounted POSIX Message Queue File System.
[   48.987404] systemd[1]: Mounted POSIX Message Queue File System.
[  OK  ] Started Increase datagram queue length.
[   49.127439] systemd[1]: Started Increase datagram queue length.
[  OK  ] Started SCW kernel requirements checker.
[   49.287114] systemd[1]: Started SCW kernel requirements checker.
[  OK  ] Started Create Static Device Nodes in /dev.
[   49.427724] systemd[1]: Started Create Static Device Nodes in /dev.
[  OK  ] Started Load Kernel Modules.
[   49.556783] systemd[1]: Started Load Kernel Modules.
[  OK  ] Started Remount Root and Kernel File Systems.
[   49.697716] systemd[1]: Started Remount Root and Kernel File Systems.
[  OK  ] Started udev Coldplug all Devices.
[   49.836791] systemd[1]: Started udev Coldplug all Devices.
[   50.000235] systemd[1]: Started Various fixups to make systemd work better on Debian.
[   50.096505] systemd[1]: Starting Load/Save Random Seed...
         Starting Load/Save Random Seed...
[   50.217859] systemd[1]: Started Rebuild Hardware Database.
[   50.283837] systemd[1]: Mounted FUSE Control File System.
[   50.350721] systemd[1]: Starting Apply Kernel Variables...
         Starting Apply Kernel Variables...
[  OK  ] Reached target Local File Systems (Pre).
[  OK  ] Reached target Local File Systems.
[  OK  ] Reached target Remote File Systems.
         Starting udev Kernel Device Manager...
[  OK  ] Listening on Syslog Socket.
         Starting Journal Service...
[  OK  ] Started udev Kernel Device Manager.
[  OK  ] Started Journal Service.
[  OK  ] Started Load/Save Random Seed.
[  OK  ] Started Apply Kernel Variables.
[  OK  ] Found device /dev/ttyS0.
[  OK  ] Created slice system-ifup.slice.
         Starting Flush Journal to Persistent Storage...
         Starting Copy rules generated while the root was ro...
[  OK  ] Started Copy rules generated while the root was ro.
[  OK  ] Started Flush Journal to Persistent Storage.
         Starting Create Volatile Files and Directories...
[  OK  ] Started Create Volatile Files and Directories.
[  OK  ] Started ifup for eth0.
         Starting ifup for eth0...
         Starting Update UTMP about System Boot/Shutdown...
[  OK  ] Reached target System Time Synchronized.
[  OK  ] Started Update UTMP about System Boot/Shutdown.
[  OK  ] Reached target System Initialization.
[  OK  ] Reached target Timers.
[  OK  ] Listening on D-Bus System Message Bus Socket.
[  OK  ] Reached target Sockets.
[  OK  ] Reached target Basic System.
         Starting LSB: Start/stop sysstat's sadc...
         Starting SCW additional volumes mounter...
         Starting /etc/rc.local Compatibility...
         Starting LSB: Start NTP daemon...
[  OK  ] Started Entropy daemon using the HAVEGE algorithm.
         Starting Entropy daemon using the HAVEGE algorithm...
         Starting SCW fetch kernel modules from Scaleway mirror...
         Starting SCW fetch ssh keys from metadata...
         Starting System Logging Service...
         Starting Login Service...
         Starting Permit User Sessions...
         Starting SCW generate ssh keys on first boot...
[  OK  ] Started Regular background program processing daemon.
         Starting Regular background program processing daemon...
         Starting LSB: Set the CPU Frequency Scaling governor to "ondemand"...
         Starting D-Bus System Message Bus...
[  OK  ] Started System Logging Service.
[  OK  ] Started LSB: Start/stop sysstat's sadc.
[  OK  ] Started /etc/rc.local Compatibility.
[  OK  ] Started LSB: Start NTP daemon.
[  OK  ] Started SCW fetch ssh keys from metadata.
[  OK  ] Started Permit User Sessions.
[  OK  ] Started LSB: Set the CPU Frequency Scaling governor to "ondemand".
[  OK  ] Started D-Bus System Message Bus.
[  OK  ] Started SCW additional volumes mounter.
[  OK  ] Started Serial Getty on ttyS0.
         Starting Serial Getty on ttyS0...
[  OK  ] Reached target Login Prompts.
[  OK  ] Started Login Service.
[  OK  ] Started SCW fetch kernel modules from Scaleway mirror.
         Starting SCW generate machine id...
[  OK  ] Started SCW generate machine id.
[  OK  ] Started SCW generate ssh keys on first boot.
[  OK  ] Started OpenBSD Secure Shell server.
         Starting OpenBSD Secure Shell server...
[  OK  ] Reached target Multi-User System.
         Starting Update UTMP about System Runlevel Changes...
[  OK  ] Started Update UTMP about System Runlevel Changes.

Ubuntu 15.04 cocky-pike ttyS0

cocky-pike login:
```

```console
root@cocky-pike:~# ps auxwww | grep getty
root      2635  0.0  0.0   3364  1632 ttyS0    Ss+  18:01   0:00 /sbin/agetty --keep-baud 115200 38400 9600 ttyS0 vt220
```

**Comment by QuentinPerez** [comment]
LGTM

**Comment by aimxhaisse** [comment]
LGTM

**Pull request merged**